### DISCUSSION TITLE
CUE as a configuration language for a streaming platform

### DISCUSSION THREAD

**Discussion by owulveryck** [cuelang/cue#1072] [Show and tell]
Hi,

I am working on a data mesh initiative. As an architect on the project, I want to prove the value of a streaming platform as part of my job.
The streaming platform is, according to me, not only an infrastructure component but needs some features such as data validation.
As a demo, I coded something with CUE and the Go SDK.
It is documented here and holds a couple of references to gists that expose the code with some tests.

https://blog.octo.com/en/pov-a-streaming-communication-platform-for-the-data-mesh

Any comment welcome.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/1072.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
